types/fs-extra/test/fs-extra-tests-module.ts

```diff
@@ -1,4 +1,4 @@
-import * as fs from 'fs-extra/esm.mjs';
+import * as fs from 'fs-extra/lib/esm.mjs';
```

Comment:
there is no 'lib', this creates non-existing paths IMO.
I was under impression that should be enough:
```json
"exports": {
    ".": {
      "types": {
        "default": "./index.d.ts"
      }
    },
    "./esm": {
      "types": {
        "default": "./esm.d.mts"
      }
    }
```

Reply:
The original module has all the sources under `lib/`.

Reply:
Yeah, you're right, this doesn't change anything. Only the exposed paths will ever work.